qiskit/ignis/verification/randomized_benchmarking/dihedral.py

```diff
@@ -520,11 +535,21 @@ def from_circuit(self, circuit):
         append_circuit(elem, circuit)
         return elem
 
-    def compose(self, other):
+    def to_matrix(self):
+        """Convert operator to Numpy matrix."""
+        return self.to_operator().data
+
+    def to_operator(self):
+        """Convert to an Operator object."""
+        return Operator(self.to_instruction())
+
+    def compose(self, other, qargs=None, front=False):
```

Comment:
Using `front=True` should be implemented to be the same as `dot`. It looks like the `qargs` element isn't implemented either, this should raise a `NotImplementedError` if it is not `None` if this is the case.